pygeoapi/provider/filesystem.py

```diff
@@ -30,6 +30,9 @@
 import io
 import logging
 import os
+from json import loads
+from pygeometa.core import read_mcf, MCFReadError
```

Comment:
Can we put the two lines of pygeometa imports at line 35?  See [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) for more information.

Reply:
can you give an example, both imports are from different modules

Reply:
```python
from json import loads
import io
import logging
import os
from urllib.parse import urljoin

from pygeometa.core import read_mcf, MCFReadError
from pygeoapi.provider.base import (BaseProvider, ProviderConnectionError,
                                    ProviderNotFoundError)
from pygeometa.schemas.stac import STACItemOutputSchema
```